README.md

```diff
@@ -1,13 +1,121 @@
 # Transition
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fchairemobilite%2Ftransition.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fchairemobilite%2Ftransition?ref=badge_shield)
 
+Transition is a modern new approach to transit planning. This repo is a Node/React web application to model, simulate and plan public transit and alternative transportation.
 
-** Soon to be open sourced **
+See http://transition.city/
 
-Transition is a modern new approach to transit planning
 
-See http://transition.city/
+[Definitions and symbols used in the code and in the interface](https://www.overleaf.com/read/dtxfhttxgjrx)
+
+For Ubuntu users: [complete step-by-step development environment setup procedure](docs/installationUbuntu20.04.md)
+
+## Non-Node Dependencies
+
+* PostgreSQL 10+ with PostGIS
```

Comment:
Wonder if the 10+ is the right version. Also maybe a version for postgis? (We might need a tested with version)

Reply:
I have no idea what Transition has been tested on! I think I had it running on 10 and 12 in the past. I'll update to 22.04 soon est I think I'll have to upgrade to 14... sounds like some doc will have to be written for it...